packages/vats/src/vat-zoe.js

```diff
@@ -6,15 +6,17 @@ const BUILD_PARAMS_KEY = 'buildZoeParams';
 export function buildRootObject(vatPowers, _vatParams, zoeBaggage) {
   const shutdownZoeVat = vatPowers.exitVatWithFailure;
 
+  let zoeConfigFacet;
+
   if (zoeBaggage.has(BUILD_PARAMS_KEY)) {
     const { feeIssuerConfig, zcfSpec } = zoeBaggage.get(BUILD_PARAMS_KEY);
-    makeDurableZoeKit({
+    ({ zoeConfigFacet } = makeDurableZoeKit({
```

Comment:
this introduces a new concept, ZoeConfigFacet, which is the return type of `makeDurableZoeKit`. 

Can we reconcile these names? 
1. call this `zoeKit` and have `getZoeKit`
2. define a new type `ZoeConfigFacet` and declare `makeDurableZoeKit` to return one of those
3. ??

I'm partial to #1 so the name matches the maker

Reply:
`makeDurableZoeKit` returns `{ zoeService, zoeConfigFacet, feeMintAccess }`, but this call only needs the `zoeConfigFacet` because the others have already been returned.

I'll add a comment.

Reply:
I misread. Thanks for the comment.